src/ApiGateway/WebsocketClient.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Bref\ApiGateway;
+
+use AsyncAws\Core\AbstractApi;
+use AsyncAws\Core\Request;
+use AsyncAws\Core\Response;
+
+class WebsocketClient extends AbstractApi
```

Comment:
From what I understand this class necessary because there is no native support in Async AWS?

If that's indeed it, could you:

- add a block comment to explain that
- add a `@internal` tag in the docblock (to say that it shouldn't be used directly by users)

Reply:
@mnapoli Correct. I'll add some docblock explaining this.